### PR TITLE
biz scenario work complete

### PR DESCRIPTION
**Pull request by LeahBineth**
Hi,
I did your business scenario. Cute idea! was clear and challenging 😃!
Hope it's all correct... Let me know if there is anything that needs correction...
just a few questions:
-    "I purchase them at wholesale for 6$ each and sell them for 12.99$ each" 
Should I add constraint on PricePurchased column that it can only be 6 and PriceSold only 12.99, or there might be other prices?
-   Report 1: you asked for a printout including the Inventory in stock, which sounds only shells that are not sold, but instructions is  also to include date sold and price sold column. Do you want to show only inventory that's not sold or all?
- Not sure what's the difference between report 1 and 2, lmk if did it wrong.

### CODE REVIEW

**Comment by cabadia26** [comment]
-Purchasing and selling these prices for now. running business on this price point. just in case wholesale market price goes up dont want system to crash. what do u suggest?
daily separate report in stock not sold and sold separate report of sold

**Comment by LeahBineth** [comment]
Right, adding a constraint that PricePurchased must be 6$ and PriceSold 12.99 $ might be too tight. It's enough to have ensure that price is greater than 0.

**Comment by cabadia26** [comment]
yes i agree on the constraint
thank u

File: BombShellDB/3 reports.sql [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+--1
+select s.style, s.color, s.size, s.datepurchased, s.pricepurchased, s.datesold, s.pricesold, PrintoutDateTime = getdate()
+from shell s
+
+--2
+select s.style, s.color, s.size, s.datepurchased, s.pricepurchased, s.datesold, s.pricesold
+from shell s
+
+--3
```
Comment (cabadia26) [inline BombShellDB/3 reports.sql:30]: --CA *in Style/Color/Size which most profit

File: BombShellDB/0 database.sql [hunk 2]
```diff
@@ -0,0 +1,8 @@
+use master
```
Comment (cabadia26) [inline BombShellDB/0 database.sql:1]: --Thank You! Excellent Work! 100% Plz see comments and no need resubmit

**Review by cabadia26 — APPROVED** [review]

**Comment by cabadia26** [comment]
--Thank You! Excellent Work! 100% Plz see comments and no need resubmit

**Pull request closed without merging**